packages/node_modules/@webex/internal-plugin-flag/src/index.js

```diff
@@ -3,26 +3,16 @@
  */
 
 import {registerInternalPlugin} from '@webex/webex-core';
+
+import '@webex/internal-plugin-conversation';
+import '@webex/internal-plugin-wdm';
 import Flag from './flag';
 import config from './config';
-import '@webex/internal-plugin-wdm';
-import {has} from 'lodash';
 
 registerInternalPlugin('flag', Flag, {
   config,
   payloadTransformer: {
-    predicates: [
-      {
-        name: 'transformObjectArray',
-        direction: 'inbound',
-        test(ctx, response) {
-          return Promise.resolve(has(response, 'body.multistatus'));
-        },
-        extract(response) {
-          return Promise.resolve(response.body.multistatus.map((item) => item && item.data && item.data.activity));
-        }
-      }
-    ],
```

Comment:
this caused web client to throw `Algorithm not allowed` error when fetching flags. It didn't like that this was defined both in flags and conversations

Reply:
If anyone can shed more light here from the sdk team that would be great. My only guess would be that the predicates are batched via the `test` and then run one after the other and if you operate on the same object twice it gets angry.